functions/packages.bash

```diff
@@ -267,9 +267,12 @@ mqtt_setup() {
   local mosquittoConf="/etc/mosquitto/mosquitto.conf"
   local mosquittoPasswd="/etc/mosquitto/passwd"
   local mqttPasswd
-  local mqttUser="openhabian"
-  local introText="The MQTT broker Eclipse Mosquitto will be installed from the official repository.\\n\\nIn addition, you can activate username:password authentication."
-  local questionText="\\nDo you want to secure your MQTT broker by a username:password combination? Every client will need to provide these upon connection.\\n\\nUsername will be '${mqttUser}', please provide a password (consisting of ASCII printable characters except space). Leave blank for no authentication, run setup again to change."
+  local mqttUser
+  local mqttDefaultUser="openhabian"
```

Comment:
shouldn't that rather be ${username:-openhabian}

Reply:
ok. done